.github/workflows/unit_tests.yml

```diff
@@ -115,38 +115,26 @@ jobs:
     #
     # 6 checks:
     # for each OS (ubuntu, macos, windows):
-    # 3.9 / mne-stable / full / validator-stable
-    # 3.12 / mne-stable / full / validator-stable
-    #
-    # 1 additional check for Apple Silicon (doesn't support Python 3.9):
+    # 3.10 / mne-stable / full / validator-stable
     # 3.12 / mne-stable / full / validator-stable
     #
     # 5 additional checks with alternative MNE-Python and BIDS validator versions:
     # ubuntu / 3.12 / mne-main / full / validator-main
-    # ubuntu / 3.9 / mne-prev / full / validator-stable
+    # ubuntu / 3.10 / mne-prev / full / validator-stable
     # ubuntu / 3.12 / mne-stable / minimal / validator-stable
     timeout-minutes: 60
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.9", "3.12"]  # Oldest and newest supported versions
+        os: [ubuntu-latest, macos-latest, macos-13, windows-latest]
```

Comment:
Based on the comments below I think this was also incorrectly specified once `macos-latest` switched from `-13` (x86_64) to `-14` (arm64). In any case might as well test both now I think